### PR TITLE
Review chapter "Caching and performance"

### PR DESCRIPTION
**Pull request by ludelafo**

### CODE REVIEW

File: 23-caching-and-performance/COURSE_MATERIAL.md
Comment (hadrylouis) [inline 23-caching-and-performance/COURSE_MATERIAL.md:1]: Perhaps I'll add the terms “cache HIT” and “cache MISS” somewhere in this document, as it seems to me that this is a term we often see in the context of caches.

File: 23-caching-and-performance/PRESENTATION.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+This will help us to improve the course and adapt the content to your needs. If
+we notice some difficulties, we will come back to you to help you.
+
+➡️ [GitHub Discussions][discussions]
+
+You can use reactions to express your opinion on a comment!
+
+## What will you do next?
+
+We are arriving at the end of the second part of the course.
```
Comment (hadrylouis) [inline 23-caching-and-performance/PRESENTATION.md:319]: third part of the course

**Review by hadrylouis — CHANGES_REQUESTED** [review]

File: 23-caching-and-performance/COURSE_MATERIAL.md [hunk 2]
```diff
@@ -0,0 +1,1029 @@
...
+   client with the new data.
+
+The request to check if the data has changed is called a **conditional
+request**.
+
+When clients want to update a resource, they can send a conditional request to
+the server to check if the data has changed since the last time it was modified.
+
+If not, the server can update the resource. If the data has changed, the server
+can return a `412 Precondition Failed` response to the client.
```
[the remark this answers is not in the document]
Comment (ludelafo) [reply 23-caching-and-performance/COURSE_MATERIAL.md]: I have rephrased this to better explain how it works. It should be better now!

File: 23-caching-and-performance/COURSE_MATERIAL.md
Comment (ludelafo) [reply 23-caching-and-performance/COURSE_MATERIAL.md:1]: I have added a Tip about these terms, good catch (the diagrams were already using _"cache hit"_/_"cache miss"_ so now it's even more clear).

File: 23-caching-and-performance/PRESENTATION.md (hunk 1 above)
Comment (ludelafo) [reply 23-caching-and-performance/PRESENTATION.md]: Thanks!

**Pull request merged**